src/utils/Processing.ts

```diff
@@ -168,6 +170,70 @@ export function writeFHIRResources(outDir: string, outPackage: Package, snapshot
   logger.info(`Exported ${count} FHIR resources as JSON.`);
 }
 
+/**
+ * Initializes an empty sample FSH within a user specified subdirectory of the current working directory
+ */
+export function init(): void {
+  logger.info(
+    'This tool will initialize a SUSHI project by creating all necessary files to build a simple IG. ' +
+      'The content of this IG will be simple default values, and should be updated. ' +
+      'The tool will now interactively create a config.yaml file for your project.'
+  );
+
+  const configDoc = YAML.parseDocument(
+    fs.readFileSync(path.join(__dirname, 'init-project', 'config.yaml'), 'utf-8')
+  );
+  // Accept user input for certain fields
+  ['name', 'id', 'canonical', 'status', 'version'].forEach(field => {
+    const userValue = readlineSync.question(
+      `${field.charAt(0).toUpperCase() + field.slice(1)} (Default: ${configDoc.get(field)}): `
+    );
+    if (userValue) {
+      configDoc.set(field, userValue);
+    }
+  });
+  // Ensure copyrightYear is accurate
+  configDoc.set('copyrightYear', `${new Date().getFullYear()}+`);
+
+  // Write init directory out, including user made config.yaml, files in utils/init-project, and build scripts from ig/files
+  const outputDir = path.resolve('.', configDoc.get('name'));
+  const initProjectDir = path.join(__dirname, 'init-project');
+  if (!readlineSync.keyInYN(`Initialize SUSHI project in ${outputDir}?`)) {
+    logger.info('Aborting initialization.');
+    return;
+  }
+
+  logger.info(`Adding files to ${outputDir}`);
```

Comment:
Again... the logging message just seems out of place here.  I'd skip it but add something at the end to signal it's done.